src/core/annotation.js

```diff
@@ -1480,6 +1491,10 @@ class WidgetAnnotation extends Annotation {
   constructor(params) {
     super(params);
 
+    if (params.needAppearances) {
+      this.appearance = null;
```

Comment:
Could we break any other code-path, e.g. related to saving or similar, if we simply clear out the `this.appearance` data here?
Since I don't know the annotation-related code well enough to tell right away, I figured that it couldn't hurt to ask.

Reply:
Normally no.
When the appearance is null, a new one should be generated from the field value..